systemtests/src/test/java/io/enmasse/systemtest/bases/TestBase.java

```diff
@@ -267,7 +267,18 @@ protected void appendAddresses(AddressSpace addressSpace, boolean wait, int batc
     protected void setAddresses(AddressSpace addressSpace, int expectedCode, Destination... destinations) throws Exception {
         TimeoutBudget budget = new TimeoutBudget(5, TimeUnit.MINUTES);
         logCollector.collectRouterState("setAddresses");
-        setAddresses(addressSpace, budget, expectedCode, destinations);
+
+        if (expectedCode == 409){
```

Comment:
please reformat the code. and rename the name of exception variable EE is not a good name for variable (use lowercase).

Reply:
Also 409 is magic number, you should use static values from java.net

Reply:
changed to ee
static value HTTP_CONFLICT used from java.net